fakebook/src/app/component/newsfeed/newsfeed.component.html

```diff
@@ -25,15 +25,12 @@ <h3>{{ user?.firstName }} {{ user?.lastName }}</h3>
     <div class="col-md-6">
       <app-new-post-form [user]="user"></app-new-post-form>
       <div *ngFor="let post of posts">
-        <app-post-view
-          [post]="post"
-          [userEmail]="user.email"
-          (notifyComment)="onNotifyComment($event)"
-        ></app-post-view>
+        <app-post-view [post]="post" [userEmail]="user.email" (notifyComment)="onNotifyComment($event)">
+        </app-post-view>
       </div>
     </div>
     <div class="col-md-6">
       <!--Insert followers component here-->
     </div>
   </div>
-</div>
+</div>
```

Comment:
these little changes like removing the newline at the end of files do not belong.